assets/shiny-server.css

```diff
@@ -0,0 +1,43 @@
+body.reconnecting {
```

Comment:
Is there a reason why `reconnecting` isn't prefaced with `ss`? I know that `disconnected` comes from Shiny.